Source/Core/DolphinQt/TAS/TASInputWindow.cpp

```diff
@@ -79,22 +81,27 @@ QGroupBox* TASInputWindow::CreateStickInputs(QString name, QSpinBox*& x_value, Q
                         .arg(name, x_shortcut_key_sequence.toString(QKeySequence::NativeText),
                              y_shortcut_key_sequence.toString(QKeySequence::NativeText)));
 
+  const int x_default = static_cast<int>(std::round(max_x / 2.));
```

Comment:
You could use `std::lround` here.
But I realize you just moved this from somewhere else.
Otherwise LGTM.

Reply:
But I would still have to cast, right? Since it returns a long and I want an int.

Reply:
Yeah, I suppose it's good as is.